src/common/logger.ts

```diff
@@ -0,0 +1,96 @@
+import * as util from "util";
+import * as vscode from "vscode";
+
+class ExtensionLogger {
```

Comment:
It would be great if we could add a short description of what this logger does and what it is used for (or to the `logger `constant below)